modules/Microsoft.RecoveryServices/vaults/replicationAlertSettings/deploy.bicep

```diff
@@ -0,0 +1,59 @@
+@description('Required. The name of the parent Azure Recovery Service Vault.')
+param recoveryVaultName string
+
+@description('Optional. The name of the replication Alert Setting.')
+param name string = 'defaultAlertSetting'
+
+@description('Optional. Comma separated list of custom email address for sending alert emails.')
+param emailAddresses array = [ '' ]
```

Comment:
For the input property names, I'd suggest in this case to stay close to the resource type property names, since they have 1:1 mapping and for maintainability reasons. In this case it would be:

- `emailAddresses `-> `customEmailAddresses`
- `emailLocale` -> `locale`
- `sendEmailToSubOwners` -> `sendToOwners`

Corresponding references should be updated in the parent module as well

Reply:
Updated the parameters to be the same as resource property names